Meadowlark-js/docker-compose.yml

```diff
@@ -188,6 +192,19 @@ services:
       - mongo-ml-local-log3:/var/log/mongodb
       - mongo-ml-local-auth:/auth
 
+  # postgres:
+  #   image: postgres:14.3-alpine@sha256:84c6ea4333ae18f25ea0fb18bb142156f2a2e545e0a779d93bbf08079e56bdaf
+  #   container_name: postgres-ml
+  #   hostname: postgres
+  #   restart: always
+  #   environment:
+  #     POSTGRES_USER: ${POSTGRES_USER:-postgres}
+  #     POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-abcdefgh1!}
+  #   ports:
+  #     - ${POSTGRES_PORT:-5432}:5432
+  #   volumes:
+  #     - pgsql-ml-data:/var/lib/postgresql/data
+
```

Comment:
Why is this commented out?

Reply:
I added the option but left it commented to avoid doing the setup every time and leave it optional